picard/ui/metadatabox/__init__.py

```diff
@@ -295,9 +295,9 @@ def get_row_info(self, row):
         return tag, value
 
     def _copy_value(self):
-        if len(self.objects) > 1:
+        if len(self.tracks) > 1 or len(self.files) > 1:
             # We cannot copy from multiple files at the same time, clear the clipboard to avoid confusion
-            log.debug("Unable to copy when multiple files are selected, clearing the clipboard")
+            log.debug("Unable to copy when multiple tracks or files are selected, clearing the clipboard")
```

Comment:
I'm not sure if this is the best logic to detect multiple selection, as I only have a vague understanding of how `tracks` and `files` relate to each other. I think that if you have a single file and single track then they are guaranteed to be related to each other, so just single selection?

Reply:
So, after last patch, I can copy tags from one track IF one file is linked to it, but not if more are linked to it, even if the tag is the same for all (I expected to not be able to copy tags that are grouped, but I expect to be able to copy those that aren't).

Reply:
I'm inclined to leave copying from multiple items (even if they have the same value) for a future PR.